config/src/main/java/com/redhat/lightblue/mongo/config/MongoConfiguration.java

```diff
@@ -238,6 +240,7 @@ public MongoClientOptions getMongoClientOptions() {
             // construct a URI to connect
             builder.socketFactory(getSocketFactory());
         }
+        builder.writeConcern(writeConcern);
```

Comment:
Don't set the write concern for all calls. Synchronization sensitive calls use the same data sources. Instead, pass the write concern to the hystrix command, and use write concern for each call.